cheroot/server.py

```diff
@@ -1037,8 +1040,9 @@ def read_request_headers(self):
             # Don't use simple_response here, because it emits headers
             # we don't want. See
             # https://github.com/cherrypy/cherrypy/issues/951
-            msg = self.server.protocol.encode('ascii')
-            msg += b' 100 Continue\r\n\r\n'
+            msg = '{} {}'.format(self.server.protocol,
+                '100 Continue\r\n\r\n') \
```

Comment:
Also, avoid escaping EOL in favor of braces where needed. Now it just violates another linter.